runtime/jcl/uma/se9_exports.xml

```diff
@@ -65,6 +65,5 @@
 	<export name="Java_java_lang_StackWalker_walkWrapperImpl" />
 	<export name="Java_java_lang_StackWalker_getImpl" />
 	<export name="Java_java_lang_invoke_MethodHandles_findNativeAddress">
-		<include-if condition="spec.flags.opt_panama and spec.flags.opt_methodHandle" />
```

Comment:
Only the panama part of the condition should be removed; it needs to be
```
    <include-if condition="spec.flags.opt_methodHandle" />
```

Reply:
I have corrected this condition.

Reply:
Actually, the export for `Java_java_lang_invoke_MethodHandles_findNativeAddress` should just be removed (like it is for cmake).

Reply:
Hi, sorry for the delay.

I have addressed this.

Reply:
The `<export>` element must be removed as a whole; removing just line 67 yields invalid XML.